examples/with-apollo-and-redux/lib/initClient.js

```diff
@@ -1,8 +1,6 @@
 import { ApolloClient, createNetworkInterface } from 'react-apollo'
 
-let apolloClient = null
-
-function _initClient (headers, initialState) {
+export const initClient = (headers, initialState = {}) => {
```

Comment:
We don't want to share the ApolloClient between different user sessions. So server-side ApolloClient needs to be created every time. So I'd leave the code intact.